src/ota.h

```diff
@@ -19,10 +19,10 @@ bool mg_ota_write(const void *buf, size_t len);  // Write chunk, aligned to 1k
 bool mg_ota_end(void);                           // Stop writing
 
 enum {
-  MG_OTA_UNAVAILABLE = 0,  // No OTA information is present
-  MG_OTA_FIRST_BOOT = 1,   // Device booting the first time after the OTA
-  MG_OTA_UNCOMMITTED = 2,  // Ditto, but marking us for the rollback
-  MG_OTA_COMMITTED = 3,    // The firmware is good
+  MG_OTA_UNAVAILABLE = 0,    // No OTA information is present
+  MG_OTA_FIRST_BOOT = 1,     // Device booting the first time after the OTA
+  MG_OTA_COMMITTED = 2,      // The firmware is good
+  MG_OTA_ROLLBACK = 4        // Rolling back to this firmware
```

Comment:
What happened to 3 ?
If those are bits I would use (1<<n) or MG_BIT()

Reply:
Yes, these are bits. I initially used MG_BIT but it didn't work since the compiler could not find the reference to MG_BIT in a certain test, I will use (1 << n) instead

Reply:
Perhaps util.h has to be included, there is one of the tests (unamalgamated ?) that honors its name